File: front/src/entity/DynamicVo.ts
import {AnswerVo} from "./AnswerVo";
import {QuestionVo} from "./QuestionVo";

export interface DynamicVo {
    eventId: number;
    userId: number;
    action: string;
    content: AnswerVo | QuestionVo;
    createTime: string;
}

File: back/src/main/java/com/qianlei/zhifou/vo/DynamicVo.java
package com.qianlei.zhifou.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

/** @author qianlei */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class DynamicVo {
  private Long eventId;
  private Integer userId;
  private String action;
  private Object content;
  private LocalDateTime createTime;
}
